docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,306 @@
+
+## Install Emulator Images
+
+If you do not have a real device, you can run applications in the Tizen emulator.
+
+To download emulator images, you can use the Tizen Package Manager or the Tizen Emulator Manager.
+
+- To use Tizen Package Manager, follow these steps:
+  1. Open **Command Palette** and enter **Run Package-Manager**.
+  2. Click **Install** to select profiles and versions that you need for your project.
+
+     ![Package Manager](media/tizen_package_manager.png)
+
+- To use Tizen Emulator Manager, follow these steps:
+
+  > **Note**  
+  >
+  > Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
+
+  1. Open **Command Palette** and enter **Run Emulator-Manager**.
+  2. Select the profiles and versions that you want to install and click **Ok**.
+
+     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
+
+
+## Develop Applications
+
+The following section explains how to use VS Code Extension for Tizen Web to develop the applications:
+
+### Create Tizen Web Project
+
+To create a Tizen Web project, follow these steps:
+
+1. Open **Command Palette** and enter **Create Wizard Project**.
+
+   ![Create Wizard Project](media/tizen_web7.png)
+
+2. Select the **Template** project type and click **Next**.
+
+   ![Select the type of project](media/tizen_web8.png)
+
+3. Select the application profile and version from the drop-down list and click Next.
+
+   ![Select the profile and version](media/tizen_web9.png)
+
+4. Select the **Basic UI** template and click **Next**.
+
+  > **Note**
+  >
+  > The list is different depending on the type and profile of the previously selected project.
+
+  ![Select the project](media/tizen_web10.png)
+
+5. Enter the **Project name** and click **Finish** to complete the project creation in workspace.
+  > **Note**
+  >
+  > packageId is an optional field, it is not mandatory to enter details in this field because the packageId will generate the details automatically.
+
+  ![Enter the name of project](media/tizen_web11.png)
+
+6. If you checked all this, the project is created.
+
+   ![Project is created](media/tizen_web12.png)
+
+### Creating Tizen Certificate
+
+To create Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Create Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > If it is created properly, profilename.p12 will be created in tizen-studio/keystore/author.
+
+To add Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Add Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > You can verify the added certificate by running **Certificate Manager**.
```

Comment:
@mijin85cho 
If you need to modify the grammar, please guide.

Reply:
@DohyungLim  I'm concerned not because of the grammar but because of the meaning difference.

**Original** : You can check the added certificate ~
**Reviewed by sabeena21** : You can verify the added certificate ~

To my knowledge the word _verify_ has the meaning of proving truth/untruth.

I think the sentence means "You can run Certificate Manager and by using it you can see the added certificate", and in this situation, "prove the added certificate" is a bit awkward to me.

@annie-samsung Would you please also see this? What is your opinion about this?

Reply:
@mijin85cho, @sabeena21 
I agree that the word "verify" changes the meaning of the sentence. The certificates authenticity is not getting verified here, it just to view the added certificate.

My recommendation would be: 
________________________
To view the added certificate, enter **Run Certificate-Manager** on the **Command Palette**.
________________________

@DohyungLim,
When I tried the create/add certificate, I found the following notification message:
**Succeed to create 'xyz' certificate** 

You can consider changing it to **'xyz' certificate is successfully created.**

I think it would be good to get the all UI messages appearing on VS code extension to be reviewed for language, so that the messages have uniform style and tone.

Reply:
Please guide me on the UI message. I'll modify it.